spec/rubocop/config_loader_spec.rb

```diff
@@ -312,18 +312,24 @@
 
       it 'unions the two lists of Excludes from the parent and child configs ' \
           'for cops that do not override the inherit_mode' do
-        excludes = configuration_from_file['Style/For']['Exclude']
-        expect(excludes.sort)
-          .to eq([File.expand_path('spec/requests/group_invite_spec.rb'),
-                  File.expand_path('spec/models/expense_spec.rb'),
-                  File.expand_path('spec/models/group_spec.rb')].sort)
+        expect do
+          excludes = configuration_from_file['Style/For']['Exclude']
+          expect(excludes.sort)
+            .to eq([File.expand_path('spec/requests/group_invite_spec.rb'),
+                    File.expand_path('spec/models/expense_spec.rb'),
+                    File.expand_path('spec/models/group_spec.rb')].sort)
+        end.to output('.rubocop.yml: Style/Dir:Exclude overrides ' \
+        "the same parameter in .rubocop_parent.yml\n").to_stderr
```

Comment:
I think we only want the warning when it's override by default. When `override` is explicitly selected for a parameter, we don't want a warning about it.

Reply:
> I think we only want the warning when it's override by default. When override is explicitly selected for a parameter, we don't want a warning about it.

Aha. I understand now. That makes perfect sense. I've addressed in c85d5f4.